packages/components/src/components/DonutChart/DonutChart.tsx

```diff
@@ -0,0 +1,101 @@
+import type { FC } from "react";
+import * as Aria from "react-aria-components";
+import clsx from "clsx";
+import styles from "./DonutChart.module.scss";
+import type { PropsWithStatus } from "@/lib/types/props";
+import type { CategoricalColors } from "@/lib/tokens/CategoricalColors";
+import { DonutChartValue } from "@/components/DonutChart/components/DonutChartValue";
+import { Donut } from "@/components/DonutChart/components/Donut";
+import { Wrap } from "@/components/Wrap";
+import { DonutChartLegend } from "@/components/DonutChart/components/DonutChartLegend";
+
+export interface DonutChartSegment {
+  value: number;
+  title: string;
+  color?: CategoricalColors;
+  valueText?: string;
+}
+
+export interface DonutChartProps
+  extends Omit<Aria.ProgressBarProps, "children" | "valueLabel">,
+    PropsWithStatus {
+  /** The size variant of the donut chart. @default "m" */
+  size?: "m" | "l";
+  /** Divides the fill of the donut chart into segments */
+  segments?: DonutChartSegment[];
+  /** An optional text to be displayed instead of the formatted value */
+  valueText?: string;
+  /**
+   * The position of the legend, if set to false, the legend is hidden @default
+   * false
+   */
+  showLegend?: "top" | "left" | "bottom" | "right" | false;
```

Comment:
This is my personal opinion, but I think it is confusing to mix the positioning and visibility. Someone not using TypeScript would be lost. Why not

```
  showLegend?: boolean;
  legendPosition?: "top" | "left" | "bottom" | "right";
  ```